client/my-sites/upgrades/components/form/hidden-input.jsx

```diff
@@ -37,14 +35,14 @@ module.exports = React.createClass( {
 	render: function() {
 		if ( this.state.toggled ) {
 			return (
-				<Input { ...this.props } initialFocus={ this.state.focus } />
-			);
-		} else {
-			return (
-				<div className="hidden-input">
-					<a href="" onClick={ this.handleClick }>{ this.props.text }</a>
-				</div>
+				<Input { ...this.props } autofocus />
 			);
 		}
+
+		return (
+			<div className="hidden-input">
+				<a href="" onClick={ this.handleClick }>{ this.props.text }</a>
+			</div>
+		);
```

Comment:
Hah, good point. I guess something like `ToggleableInput` (unless "toggleable" is not actually a word :stuck_out_tongue_winking_eye:) would be a better name.